src/web/shared/Contours.ts

```diff
@@ -4,7 +4,7 @@ import { ProjectionLike } from "ol/proj";
 
 const equirectangular = "EPSG:4326";
 
-export function geoJSONToDepthContourFeatures(projection: ProjectionLike, geojson: object) {
+export function geoJSONToDepthContourFeatures(projection: ProjectionLike, geojson: any) {
```

Comment:
Since this function was modified and does not have a function comment yet we should add one.